rossum/lib/api_client.py

```diff
@@ -357,8 +357,10 @@ def _get_annotation_polling(self, annotation: int) -> dict:
 
     def create_workspace(
         self, name: str, organization: str, metadata: Optional[Dict] = None
-    ) -> dict:
-        data = {"name": name, "organization": organization, "metadata": metadata}
+    ) -> Dict[str, Any]:
+        data = {"name": name, "organization": organization}  # type: Dict[str, Any]
+        if metadata:
+            data.update({"metadata": metadata})
```

Comment:
Please add a test testing all rows of this function to `class TestRossumClient:`.
As we are moving away from supporting `rossum` as a command line tool to a Python API Client, we should also put more focus on testing the new functionalities added to `api_client.py`